assignment/index.html

```diff
@@ -70,10 +70,43 @@
 
       ===================== */
 
-      var jsonToCsv = function(json) { console.log(json); };
-
+      var jsonToCsv = function(json) { 
+        var csvArray=[]
+        csvArray[0] = Object.keys(json[0])
+        json.forEach(function(item,index) {
+          if(index>=1){
+            csvArray[index] = Object.values(item)
+          }
+        });
```

Comment:
This is clever, but you might prefer one of these strategies to simply cut the head of (take the tail) of your array:
`const [head, ...tail] = arr;`
`var tail = arr.slice(1);`